src/ClearingHouse.sol

```diff
@@ -249,15 +270,7 @@ contract ClearingHouse is Policy, RolesConsumer, CoolerCallback {
     function defund(ERC20 token_, uint256 amount_) external onlyRole("cooler_overseer") {
```

Comment:
Is the `amount_` parameter in terms of DAI or sDAI? Would be helpful to clarify.

Reply:
in this case, this is a generic-purpose function to transfer any ERC20 back to the treasury, so `amount_` will be in `token_` terms.